AWSSDKSwiftCLI/Sources/SPR/VerifyPackage.swift

```diff
@@ -0,0 +1,31 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import AWSCLIUtils
+
+extension SPRPublisher {
+
+    func verifyPackage() throws {
```

Comment:
This function:
- Gets the package description for the package being published
- Verifies that the name being published matches the name in the package description.